mono/utils/checked-build.h

```diff
@@ -201,10 +201,14 @@ void check_metadata_store_local(void *from, void *to);
 
 void checked_build_thread_transition(const char *transition, void *info, int from_state, int suspend_count, int next_state, int suspend_count_delta);
 
+void g_error_with_history(const char *msg, ...);
```

Comment:
We should avoid the `g_` prefix in general. We use it in eglib because we remap all symbols to `eg_...`, but we wouldn't be doing that here, so it could potentially clash with some function in glib in the future (though unlikely, granted).